### PR TITLE
docs: adds FAQ

### PR DESCRIPTION
**Pull request by kendallstrautman**
Had some merge issues with this [older PR](https://github.com/tinacms/tinacms.org/pull/546).

---
 _Copied Notes_

Hey all, I added a ton of new questions and answers. I expect the answers to need a lot of work / massaging, but I just wanted to throw paint on the wall to get the conversations churning. I copied / shaped some answers based on responses in the forum. Please, rip it up and change the answers / remove questions as you think best, I'm sure there are things I misrepresented or didn't explain well.

But let's get this through the door together 👌

### CODE REVIEW

File: content/docs/getting-started/faq.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+- Another would be to completely rewrite the editing UI with Vue components.
+- A third option would be to convert the editing UI to [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components), so they can be independent of a UI framework.
+
+The Core Tina Team is focused on creating a solid foundation with React first, before considering Vue. However, if someone in the community wants to spearhead this effort, we will certainly offer support and guidance.
+
+## Who should use Tina?
+
+From a **development perspective**, folks who are comfortable with React and Javascript.
+
+From a **content editing perspective**, the UX has been created with developers-first in mind. For example, the [GitHub editing workflow](https://tinacms.org/guides/nextjs/github/initial-setup) and interface assumes the content editor is comfortable with Git terminology such as forking, branching, pull requests etc. We chose to focus on developers editing sites before adding abstractions to smooth out the experience for non-technical editors.
```
Comment (ncphillips) [inline content/docs/getting-started/faq.md:47]:
```suggestion
From a **content editing perspective**, the current UX has been created with developers-first in mind. For example, the [GitHub editing workflow](https://tinacms.org/guides/nextjs/github/initial-setup) and interface assumes the content editor is comfortable with Git terminology such as forking, branching, pull requests etc. We chose to focus on developers editing sites before adding abstractions to smooth out the experience for non-technical editors. This strategy has allowed us to quickly expand Tina's core capabilities and improve it's stability.
```

Comment (ncphillips) [reply content/docs/getting-started/faq.md]: Added the last sentence and the word "current" in "the current UX has been crafted.

My main concern is emphasizing that this is not the final UX and that Tina is intended to be for everyone, but we need to take baby steps to get there.

File: content/docs/getting-started/faq.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+title: Frequently Asked Questions
+---
+
+We hope these answers give you an honest insight into the project — its current state, the possibilities, and supported usecases etc. Please share additional questions on the [community forum](https://community.tinacms.org/)!
+
+## Why do you say "Tina is not a CMS?"
```
Comment (ncphillips) [inline content/docs/getting-started/faq.md:7]: Sick 👍

File: content/docs/getting-started/faq.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- **Headless Paradigm:** A CMS is an application that let's you edit your content. It lives independent of your website. Your website's build process loads data from the CMS to create the web pages. Your website and the content management are completely independent.
+- **The Tina Paradigm:** Your website is it's own CMS. You use Tina to build content editing capabilities directly into your website interface. You can set up a custom or pre-configured backend to handle how and where content is stored.
+
+Tina is not an application on its own that you login to, however you could build that authenticated editing application into your site with Tina. It is not a one-size-fits all solution. It is incredibly flexible, providing precise control over an entire content editing infrastructure — from the database to the user experience.
+
+## Is Tina stable?
+
+Currently, Tina is pre Version-1. While much of the core architecture is fairly-well settled, many of the higher-level APIs are still being built and shaped. Breaking changes in non-core packages are introduced regularly (but communicated in Release Notes) as we push limits and hone the developer experience.
```
Comment (kendallstrautman) [inline content/docs/getting-started/faq.md:23]:
```suggestion
Currently, Tina is pre Version-1. While much of the core architecture is fairly-well settled, many of the higher-level APIs are still being built and shaped. Breaking changes in non-core packages are introduced regularly (but communicated in [Release Notes](/docs/releases)) as we push limits and hone the developer experience.
```

File: content/docs/getting-started/faq.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+While it is entirely possible to use Tina with Vue, currently there are no adaptors, packages, or documented methods of doing this. This is an unbeaten path.
+
+We can identify a few different ways to approach Vue support:
+
+- One way would be to create an adaptor that allows you to set up the React editing UI within your Vue site. This isn't ideal for many reasons, but it is an option.
+- Another would be to completely rewrite the editing UI with Vue components.
+- A third option would be to convert the editing UI to [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components), so they can be independent of a UI framework.
+
+The Core Tina Team is focused on creating a solid foundation with React first, before considering Vue. However, if someone in the community wants to spearhead this effort, we will certainly offer support and guidance.
```
Comment (kendallstrautman) [inline content/docs/getting-started/faq.md:41]:
```suggestion
The Core Tina Team is focused on creating a solid foundation with React first, before considering Vue. However, if someone in the community wants to [spearhead this effort](https://github.com/tinacms/tinacms/issues/1258), we will certainly offer support and guidance.
```

File: content/docs/getting-started/faq.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+## What about Vue?
+
+While it is entirely possible to use Tina with Vue, currently there are no adaptors, packages, or documented methods of doing this. This is an unbeaten path.
+
+We can identify a few different ways to approach Vue support:
+
+- One way would be to create an adaptor that allows you to set up the React editing UI within your Vue site. This isn't ideal for many reasons, but it is an option.
+- Another would be to completely rewrite the editing UI with Vue components.
+- A third option would be to convert the editing UI to [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components), so they can be independent of a UI framework.
```
Comment (kendallstrautman) [inline content/docs/getting-started/faq.md:39]:
```suggestion
- A third option would be to convert the editing UI to [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) (with [something like Stencil](https://github.com/tinacms/tinacms/pull/601)), so they can be independent of a UI framework.
```

File: content/docs/getting-started/faq.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+---
+title: Frequently Asked Questions
+---
+
+We hope these answers give you an honest insight into the project — its current state, the possibilities, and supported usecases etc. Please share additional questions on the [community forum](https://community.tinacms.org/)!
+
+## Why do you say "Tina is not a CMS?"
+
+Tina introduces an entirely new paradigm to the content management space, which can make it difficult to grasp. In short, Tina is a toolkit for making your website it's own CMS. It's a suite of packages that enables developers to build a customized content management system into the website itself.
```
Comment (DirtyF) [inline content/docs/getting-started/faq.md:9]:
```suggestion
Tina introduces an entirely new paradigm to the content management space, which can make it difficult to grasp. In short, Tina is a toolkit for making your website its own CMS. It's a suite of packages that enables developers to build a customized content management system into the website itself.
```

File: content/docs/getting-started/faq.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+Tina allows you to expose content editing directly within your website's components. You can use Tina to build your own custom CMS and have precise control over how editors interact with those editing capabilities.
+
+Here's an overview of the current CMS landscape:
+
+- **Traditional Paradigm:** A CMS is an application you use to build a website (i.e. Wordpress, Drupal, etc.). A conventional CMS provides a fully-realized editing interface, coupled with a backend for managing data, requiring you to create and edit content within its bounds. You build your website on top of the CMS.
+- **Headless Paradigm:** A CMS is an application that let's you edit your content. It lives independent of your website. Your website's build process loads data from the CMS to create the web pages. Your website and the content management are completely independent.
+- **The Tina Paradigm:** Your website is it's own CMS. You use Tina to build content editing capabilities directly into your website interface. You can set up a custom or pre-configured backend to handle how and where content is stored.
```
Comment (DirtyF) [inline content/docs/getting-started/faq.md:17]:
```suggestion
- **The Tina Paradigm:** Your website is its own CMS. You use Tina to build content editing capabilities directly into your website interface. You can set up a custom or pre-configured backend to handle how and where content is stored.
```

File: content/docs/getting-started/faq.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+## Is Tina stable?
+
+Currently, Tina is pre Version-1. While much of the core architecture is fairly-well settled, many of the higher-level APIs are still being built and shaped. Breaking changes in non-core packages are introduced regularly (but communicated in [Release Notes](/docs/releases)) as we push limits and hone the developer experience.
+
+## What framework do you best support?
+
+Tina's editing UI is built entirely in React. Right now, you can only use Tina in React-based environments, such as [CRA](https://reactjs.org/docs/create-a-new-react-app.html), [Next.js](https://nextjs.org/), or [Gatsby](https://www.gatsbyjs.com/).
+
+As of now, we'd recommend using CRA or Next.js if you're wanting to use [Tina in production](/docs/getting-started/faq#what-is-the-editing-environment-how-does-tina-work-in-production) due to a few constraints in the way Gatsby works.
```
Comment (DirtyF) [inline content/docs/getting-started/faq.md:29]:
```suggestion
As of now, we'd recommend using Create React App (CRA) or Next.js if you're wanting to use [Tina in production](/docs/getting-started/faq#what-is-the-editing-environment-how-does-tina-work-in-production) due to a few constraints in the way Gatsby works.
```

File: content/docs/getting-started/faq.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+- One way would be to create an adaptor that allows you to set up the React editing UI within your Vue site. This isn't ideal for many reasons, but it is an option.
+- Another would be to completely rewrite the editing UI with Vue components.
+- A third option would be to convert the editing UI to [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) (with [something like Stencil](https://github.com/tinacms/tinacms/pull/601)), so they can be independent of a UI framework.
+
+The Core Tina Team is focused on creating a solid foundation with React first, before considering Vue. However, if someone in the community wants to [spearhead this effort](https://github.com/tinacms/tinacms/issues/1258), we will certainly offer support and guidance.
+
+## Who should use Tina?
+
+From a **development perspective**, folks who are comfortable with React and Javascript.
```
Comment (DirtyF) [inline content/docs/getting-started/faq.md:45]:
```suggestion
From a **development perspective**, folks who are comfortable with React and JavaScript.
```

**Pull request merged**